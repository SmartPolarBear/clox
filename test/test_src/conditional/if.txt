R"(
var num=(10);
if(num==11)
{
    print "fuck";
}
else
{
    print "great";
}

print num==11?"fuck":(num==10?"yes":"no");

{
    var num=nil;
    if(num)
    {
        print "fuck";
    }
    else
    {
        print "great";
    }
}

var str="xkzgreat";
if(str=="xkzgreat")
{
    print "yes, xkz is great!";
}
else
{
    print "fuck, you are wrong.";
}

print str=="xkzgreat"?"he is really great!":"fuck!";
)"