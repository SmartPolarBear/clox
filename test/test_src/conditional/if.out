R"(great
yes
great
yes, xkz is great!
he is really great!)"